backend/src/main/java/ai/verta/modeldb/utils/ModelDBUtils.java

```diff
@@ -26,12 +26,7 @@
 import ai.verta.uac.GetCollaboratorResponse;
 import ai.verta.uac.ShareViaEnum;
 import ai.verta.uac.UserInfo;
-import com.google.protobuf.Any;
-import com.google.protobuf.GeneratedMessageV3;
-import com.google.protobuf.InvalidProtocolBufferException;
-import com.google.protobuf.Message;
-import com.google.protobuf.MessageOrBuilder;
-import com.google.protobuf.Value;
+import com.google.protobuf.*;
```

Comment:
we keep undoing , redoing these changes , if these are automated can both @evgeniilezhnin  and @ad-47  get to a consensus and update their editor settings.
I like expanding the *.

Reply:
sorry, I was reinstalling idea recently and forgot to set this parameter to expand *. Now I think I have fixed it